### PR TITLE
Limit Cog1 to require at least 14 players on RP server

### PR DESCRIPTION
**Pull request by MarkNais**
[REWORK][FEEDBACK]
## About the PR 
On the RP server: ~~20~~ 14 players must be connected to be able to vote for Cog1.

## Why's this needed? 
Cog1 may be comfortable, but it does not help create RP situations.

- Some departments have little/no hallway-facing rooms. Notably Science, and to a lesser extent Botany (An empty reception desk doesn't count i.m.o)
- There are massive stretches of hallway that are (on low pop) just walking space. 
  - From Science to Security, there's nothing worth stopping for.
  - From medbay to cargo, the only notable "stop" is mechanics.

Maps like Clarion and Manta strike a good balance of exposing departments without needing too much hallway space.

## Changelog

```
(u)MarkNstein:
(+)On the RP server: Cogmap 1 can only be voted for when at least 14 players are connected.
```

### CODE REVIEW

**Comment by MarkNais** [comment]
As a less formal note: I've spent a few rounds adding graviton accelerators to Cog1 and Kondaru in order to make up for these short comings, but this should not be expected of players.

**Comment by Ley-bunned** [comment]
Yes, this is sorely needed. Particularly if you've tried to play RP with 6 people on cog1.

**Comment by KnellerUni** [comment]
I think 20 is too high of a limit. I've had decent games on Cogmap with 10-20 players, and limiting one of the most popular maps to when there are *at least 20* players (which let's face it, isn't as often as we'd like) isn't going to make many people happy. I think the limit should be set at minimum 14. At 14, you can be pretty sure you've got at least 2 people in each Department, and a few Civs wandering about.

File: code/map.dm [hunk 1]
```diff
@@ -12,7 +12,11 @@ var/global/datum/map_settings/map_settings = null
 //playerPickable defines whether the map can be chosen by players when voting on a new map. Setting to ASS_JAM should allow it on the 13th only, and not on RP.
 var/global/list/mapNames = list(
 	"Clarion" = 		list("id" = "CLARION", 		"settings" = "destiny/clarion", "playerPickable" = 1),
+#ifdef RP_MODE
+	"Cogmap 1" = 		list("id" = "COGMAP", 		"settings" = "cogmap", 			"playerPickable" = 1, 	"MinPlayersAllowed" = 20),
```
Comment (MarkNais) [inline code/map.dm:16]:
```suggestion
	"Cogmap 1" = 		list("id" = "COGMAP", 		"settings" = "cogmap", 			"playerPickable" = 1, 	"MinPlayersAllowed" = 14),
```

**Comment by LadyBirb** [comment]
yes merge please

**Comment by Kubius** [comment]
Bit of a side thought, but figured it'd probably be worth mentioning: before merge, the changelog in the PR description should be updated to reflect whatever number is settled on.

**Pull request merged**